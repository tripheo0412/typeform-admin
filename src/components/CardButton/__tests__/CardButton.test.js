import React from 'react';
import { shallow } from 'enzyme';
import { Theme, Template } from '..';

describe('Card Button Test', () => {
  it('renders Theme without crashing', () => {
    shallow(<Theme />);
  });
  it('renders Template without crashing', () => {
    shallow(<Template />);
  });
  it('should render correctly default template', () => {
    const component = shallow(<Theme />);
    expect(component).toMatchSnapshot();
  });
  it('should render card text correctly with responses', () => {
    const component = shallow(<Theme responses={4} />);
    expect(component).toMatchSnapshot();
  });
  it('should render card text correctly without responses', () => {
    const component = shallow(<Theme responses={0} />);
    expect(component).toMatchSnapshot();
  });
  it('should render Question card correctly', () => {
    const component = shallow(<Theme isSample tempColor="red" />);
    expect(component).toMatchSnapshot();
  });
});
