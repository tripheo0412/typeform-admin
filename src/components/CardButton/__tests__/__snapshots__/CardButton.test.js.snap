// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Card Button Test should render Question card correctly 1`] = `ShallowWrapper {}`;

exports[`Card Button Test should render card text correctly with responses 1`] = `ShallowWrapper {}`;

exports[`Card Button Test should render card text correctly without responses 1`] = `ShallowWrapper {}`;

exports[`Card Button Test should render correctly default template 1`] = `ShallowWrapper {}`;
